src/name.ts

```diff
@@ -265,6 +265,18 @@ export class Name {
           } else {
             if (typeId === this.getName("E") || typeId === this.getName("DC")) {
               asnAttr.value.ia5String = value;
+            } else if (typeof value === "object") {
+                const entry = value as object;
```

Comment:
Typescript compiler seemed to want this cast conversion, even though the `typeof` check above ensures it.

Reply:
I guess the reason is that `typeof value === "object"` will be true for value = null.
So you may add a null check to the if statement, or process this case separately, to ensure the null value case is handled correctly.

Reply:
Adjusted JsonAttributeAndValue and order of operations for stricter type control.